### PR TITLE
[Bots][1700920][Steps correction]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Corrected the process steps. Please review.

### CODE REVIEW

File: msteams-platform/bots/how-to/authentication/add-authentication.md [hunk 1]
```diff
@@ -127,8 +127,10 @@ Note-there are two options for Service Providers here-Azure AD V1 and Azure AD V
 
 1. In the [**Azure portal**][azure-portal], select your resource group from the dashboard.
 1. Select your bot channel registration link.
-1. On the resource page, select **Settings**.
-1. Under **OAuth Connection Settings** near the bottom of the page, select **Add Setting**.
+1. Open the resource page and select **Configuration** under **Settings**. 
+1. Select **Add OAuth Connection Settings** near the bottom of the page.
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/authentication/add-authentication.md:131]:
```suggestion
1. Select **Add OAuth Connection Settings**.  
```

File: msteams-platform/bots/how-to/authentication/add-authentication.md [hunk 2]
```diff
@@ -154,8 +156,11 @@ Note-there are two options for Service Providers here-Azure AD V1 and Azure AD V
...
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/authentication/add-authentication.md:160]:
```suggestion
1. Select **Add OAuth Connection Settings**.  
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Rajeshwari-v - please commit the suggestions asap and add Wajeed as reviewer.

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by Wajeed-msft — APPROVED** [review]
Approved.

**Pull request merged**